src/github_runner_image_builder/cli.py

```diff
@@ -115,3 +115,89 @@ def run(  # pylint: disable=too-many-arguments
     if callback_script:
         # The callback script is a user trusted script.
         subprocess.check_call([str(callback_script), image_id])  # nosec: B603
+
+
+@main.command(name="init_external")
+@click.argument("cloud_name")
+def initialize_external(cloud_name: str) -> None:
+    """Initialize builder CLI function wrapper.
+
+    Args:
+        cloud_name: The cloud to use from the clouds.yaml file. The CLI looks for clouds.yaml in
+            paths of the following order: current directory, ~/.config/openstack, /etc/openstack.
+    """
+    arch = get_supported_arch()
+    openstack_builder.initialize(arch=arch, cloud_name=cloud_name)
+
+
+@main.command(name="experimental-run-external")
+@click.argument("cloud_name")
+@click.argument("image_name")
+@click.argument("flavor")
+@click.argument("network")
+@click.option(
+    "-b",
+    "--base-image",
+    type=click.Choice(BASE_CHOICES),
+    default="noble",
+    help=("The Ubuntu base image to use as build base."),
+)
+@click.option(
+    "-k",
+    "--keep-revisions",
+    default=5,
+    help="The maximum number of images to keep before deletion.",
+)
+@click.option(
+    "-s",
+    "--callback-script",
+    type=click.Path(exists=True),
+    default=None,
+    help=(
+        "The callback script to trigger after image is built. The callback script is called"
+        "with the first argument as the image ID."
+    ),
+)
+@click.option(
+    "-r",
```

Comment:
I've given it a bit of thought:
`-v`: sounds like it would be version or verbose
`-rv`: I don't think is suitable because usually shorthands should be 1 character, if more than 1 char is used, it is regarded as two separate shorthands (`-r` `-v`).
Would keeping `r` be okay?

Reply:
What about removing the shortened switch altogether if we can't come up with a good shorthand?